<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Morphologically similar, but regionally distinct: Perdiz arrow points from Caddo burial contexts in the American Southeast</title>
  <meta name="description" content="Morphological trajectories and disparity associated with resharpening." />
  <meta name="generator" content="bookdown 0.25 and GitBook 2.6.7" />

  <meta property="og:title" content="Morphologically similar, but regionally distinct: Perdiz arrow points from Caddo burial contexts in the American Southeast" />
  <meta property="og:type" content="book" />
  <meta property="og:image" content="http://seldenlab.github.io/perdiz.4/img/img2/trajectory.png" />
  <meta property="og:description" content="Morphological trajectories and disparity associated with resharpening." />
  <meta name="github-repo" content="seldenlab/perdiz.4" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Morphologically similar, but regionally distinct: Perdiz arrow points from Caddo burial contexts in the American Southeast" />
  
  <meta name="twitter:description" content="Morphological trajectories and disparity associated with resharpening." />
  <meta name="twitter:image" content="http://seldenlab.github.io/perdiz.4/img/img2/trajectory.png" />

<meta name="author" content="Robert Z. Selden, Jr." />


<meta name="date" content="2022-04-06" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  

<link rel="next" href="linear-shapesize.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="custom.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">perdiz.4</a></li>

<li class="divider"></li>
<li><a href="index.html#preface">Preface<span></span></a>
<ul>
<li class="chapter" data-level="0.1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>0.1</b> Primary findings<span></span></a>
<ul>
<li class="chapter" data-level="0.1.1" data-path="index.html"><a href="index.html#linear-metrics"><i class="fa fa-check"></i><b>0.1.1</b> Linear metrics<span></span></a></li>
<li class="chapter" data-level="0.1.2" data-path="index.html"><a href="index.html#geometric-morphometrics"><i class="fa fa-check"></i><b>0.1.2</b> Geometric morphometrics<span></span></a></li>
</ul></li>
<li class="chapter" data-level="0.2" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i><b>0.2</b> Acknowledgments<span></span></a></li>
<li class="chapter" data-level="0.3" data-path="index.html"><a href="index.html#funding"><i class="fa fa-check"></i><b>0.3</b> Funding<span></span></a></li>
<li class="chapter" data-level="0.4" data-path="index.html"><a href="index.html#data-management"><i class="fa fa-check"></i><b>0.4</b> Data management<span></span></a></li>
<li class="chapter" data-level="0.5" data-path="index.html"><a href="index.html#colophon"><i class="fa fa-check"></i><b>0.5</b> Colophon<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="linear-shapesize.html"><a href="linear-shapesize.html"><i class="fa fa-check"></i><b>1</b> Linear ~ shape/size<span></span></a>
<ul>
<li class="chapter" data-level="1.1" data-path="linear-shapesize.html"><a href="linear-shapesize.html#load-packages-data"><i class="fa fa-check"></i><b>1.1</b> Load packages + data<span></span></a></li>
<li class="chapter" data-level="1.2" data-path="linear-shapesize.html"><a href="linear-shapesize.html#generalised-procrustes-analysis"><i class="fa fa-check"></i><b>1.2</b> Generalised Procrustes Analysis<span></span></a></li>
<li class="chapter" data-level="1.3" data-path="linear-shapesize.html"><a href="linear-shapesize.html#bpls-maximum-length"><i class="fa fa-check"></i><b>1.3</b> 2BPLS Maximum length<span></span></a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="linear-shapesize.html"><a href="linear-shapesize.html#shape"><i class="fa fa-check"></i><b>1.3.1</b> Shape<span></span></a></li>
<li class="chapter" data-level="1.3.2" data-path="linear-shapesize.html"><a href="linear-shapesize.html#size"><i class="fa fa-check"></i><b>1.3.2</b> Size<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="linear-shapesize.html"><a href="linear-shapesize.html#bpls-maximum-blade-length"><i class="fa fa-check"></i><b>1.4</b> 2BPLS Maximum blade length<span></span></a>
<ul>
<li class="chapter" data-level="1.4.1" data-path="linear-shapesize.html"><a href="linear-shapesize.html#shape-1"><i class="fa fa-check"></i><b>1.4.1</b> Shape<span></span></a></li>
<li class="chapter" data-level="1.4.2" data-path="linear-shapesize.html"><a href="linear-shapesize.html#size-1"><i class="fa fa-check"></i><b>1.4.2</b> Size<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="linear-shapesize.html"><a href="linear-shapesize.html#bpls-maximum-shoulder-width"><i class="fa fa-check"></i><b>1.5</b> 2BPLS Maximum shoulder width<span></span></a>
<ul>
<li class="chapter" data-level="1.5.1" data-path="linear-shapesize.html"><a href="linear-shapesize.html#shape-2"><i class="fa fa-check"></i><b>1.5.1</b> Shape<span></span></a></li>
<li class="chapter" data-level="1.5.2" data-path="linear-shapesize.html"><a href="linear-shapesize.html#size-2"><i class="fa fa-check"></i><b>1.5.2</b> Size<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="linear-shapesize.html"><a href="linear-shapesize.html#bpls-maximum-width"><i class="fa fa-check"></i><b>1.6</b> 2BPLS Maximum width<span></span></a>
<ul>
<li class="chapter" data-level="1.6.1" data-path="linear-shapesize.html"><a href="linear-shapesize.html#shape-3"><i class="fa fa-check"></i><b>1.6.1</b> Shape<span></span></a></li>
<li class="chapter" data-level="1.6.2" data-path="linear-shapesize.html"><a href="linear-shapesize.html#size-3"><i class="fa fa-check"></i><b>1.6.2</b> Size<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1.7" data-path="linear-shapesize.html"><a href="linear-shapesize.html#bpls-maximum-thickness"><i class="fa fa-check"></i><b>1.7</b> 2BPLS Maximum thickness<span></span></a>
<ul>
<li class="chapter" data-level="1.7.1" data-path="linear-shapesize.html"><a href="linear-shapesize.html#shape-4"><i class="fa fa-check"></i><b>1.7.1</b> Shape<span></span></a></li>
<li class="chapter" data-level="1.7.2" data-path="linear-shapesize.html"><a href="linear-shapesize.html#size-4"><i class="fa fa-check"></i><b>1.7.2</b> Size<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1.8" data-path="linear-shapesize.html"><a href="linear-shapesize.html#bpls-maximum-stem-length"><i class="fa fa-check"></i><b>1.8</b> 2BPLS Maximum stem length<span></span></a>
<ul>
<li class="chapter" data-level="1.8.1" data-path="linear-shapesize.html"><a href="linear-shapesize.html#shape-5"><i class="fa fa-check"></i><b>1.8.1</b> Shape<span></span></a></li>
<li class="chapter" data-level="1.8.2" data-path="linear-shapesize.html"><a href="linear-shapesize.html#size-5"><i class="fa fa-check"></i><b>1.8.2</b> Size<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1.9" data-path="linear-shapesize.html"><a href="linear-shapesize.html#bpls-maximum-stem-width"><i class="fa fa-check"></i><b>1.9</b> 2BPLS Maximum stem width<span></span></a>
<ul>
<li class="chapter" data-level="1.9.1" data-path="linear-shapesize.html"><a href="linear-shapesize.html#shape-6"><i class="fa fa-check"></i><b>1.9.1</b> Shape<span></span></a></li>
<li class="chapter" data-level="1.9.2" data-path="linear-shapesize.html"><a href="linear-shapesize.html#size-6"><i class="fa fa-check"></i><b>1.9.2</b> Size<span></span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="linear-metrics-1.html"><a href="linear-metrics-1.html"><i class="fa fa-check"></i><b>2</b> Linear metrics<span></span></a>
<ul>
<li class="chapter" data-level="2.1" data-path="linear-metrics-1.html"><a href="linear-metrics-1.html#derived-variables"><i class="fa fa-check"></i><b>2.1</b> Derived variables<span></span></a></li>
<li class="chapter" data-level="2.2" data-path="linear-metrics-1.html"><a href="linear-metrics-1.html#assign-size-classes"><i class="fa fa-check"></i><b>2.2</b> Assign <code>size classes</code><span></span></a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="linear-metrics-1.html"><a href="linear-metrics-1.html#northern-behavioural-region"><i class="fa fa-check"></i><b>2.2.1</b> Northern behavioural region<span></span></a></li>
<li class="chapter" data-level="2.2.2" data-path="linear-metrics-1.html"><a href="linear-metrics-1.html#southern-behavioural-region"><i class="fa fa-check"></i><b>2.2.2</b> Southern behavioural region<span></span></a></li>
<li class="chapter" data-level="2.2.3" data-path="linear-metrics-1.html"><a href="linear-metrics-1.html#aggregate"><i class="fa fa-check"></i><b>2.2.3</b> Aggregate<span></span></a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="linear-metrics-1.html"><a href="linear-metrics-1.html#shape-variables"><i class="fa fa-check"></i><b>2.3</b> Shape variables<span></span></a></li>
<li class="chapter" data-level="2.4" data-path="linear-metrics-1.html"><a href="linear-metrics-1.html#ratios"><i class="fa fa-check"></i><b>2.4</b> Ratios<span></span></a></li>
<li class="chapter" data-level="2.5" data-path="linear-metrics-1.html"><a href="linear-metrics-1.html#boxplots-for-size-class"><i class="fa fa-check"></i><b>2.5</b> Boxplots for <code>Size Class</code><span></span></a></li>
<li class="chapter" data-level="2.6" data-path="linear-metrics-1.html"><a href="linear-metrics-1.html#principal-components-analysis"><i class="fa fa-check"></i><b>2.6</b> Principal Components Analysis<span></span></a></li>
<li class="chapter" data-level="2.7" data-path="linear-metrics-1.html"><a href="linear-metrics-1.html#analyses-of-variance-anova"><i class="fa fa-check"></i><b>2.7</b> Analyses of Variance (ANOVA)<span></span></a>
<ul>
<li class="chapter" data-level="2.7.1" data-path="linear-metrics-1.html"><a href="linear-metrics-1.html#maximum-blade-length-size-class"><i class="fa fa-check"></i><b>2.7.1</b> <em>Maximum blade length</em> ~ <em><code>size class</code></em><span></span></a></li>
<li class="chapter" data-level="2.7.2" data-path="linear-metrics-1.html"><a href="linear-metrics-1.html#maximum-shoulder-width-size-class"><i class="fa fa-check"></i><b>2.7.2</b> <em>Maximum shoulder width</em> ~ <em><code>size class</code></em><span></span></a></li>
<li class="chapter" data-level="2.7.3" data-path="linear-metrics-1.html"><a href="linear-metrics-1.html#maximum-width-size-class"><i class="fa fa-check"></i><b>2.7.3</b> <em>Maximum width</em> ~ <em><code>size class</code></em><span></span></a></li>
<li class="chapter" data-level="2.7.4" data-path="linear-metrics-1.html"><a href="linear-metrics-1.html#maximum-thickness-size-class"><i class="fa fa-check"></i><b>2.7.4</b> <em>Maximum thickness</em> ~ <em><code>size class</code></em><span></span></a></li>
<li class="chapter" data-level="2.7.5" data-path="linear-metrics-1.html"><a href="linear-metrics-1.html#maximum-stem-length-size-class"><i class="fa fa-check"></i><b>2.7.5</b> <em>Maximum stem length</em> ~ <em><code>size class</code></em><span></span></a></li>
<li class="chapter" data-level="2.7.6" data-path="linear-metrics-1.html"><a href="linear-metrics-1.html#maximum-stem-width-size-class"><i class="fa fa-check"></i><b>2.7.6</b> <em>Maximum stem width</em> ~ <em><code>size class</code></em><span></span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="geometric-morphometrics-1.html"><a href="geometric-morphometrics-1.html"><i class="fa fa-check"></i><b>3</b> Geometric morphometrics<span></span></a>
<ul>
<li class="chapter" data-level="3.1" data-path="geometric-morphometrics-1.html"><a href="geometric-morphometrics-1.html#load-packages-data-1"><i class="fa fa-check"></i><b>3.1</b> Load packages + data<span></span></a></li>
<li class="chapter" data-level="3.2" data-path="geometric-morphometrics-1.html"><a href="geometric-morphometrics-1.html#generalised-procrustes-analysis-1"><i class="fa fa-check"></i><b>3.2</b> Generalised Procrustes Analysis<span></span></a></li>
<li class="chapter" data-level="3.3" data-path="geometric-morphometrics-1.html"><a href="geometric-morphometrics-1.html#boxplot-centroid-size"><i class="fa fa-check"></i><b>3.3</b> Boxplot (centroid size)<span></span></a></li>
<li class="chapter" data-level="3.4" data-path="geometric-morphometrics-1.html"><a href="geometric-morphometrics-1.html#principal-components-analysis-1"><i class="fa fa-check"></i><b>3.4</b> Principal Components Analysis<span></span></a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="geometric-morphometrics-1.html"><a href="geometric-morphometrics-1.html#minimamaxima-of-pc12-with-warp-grids"><i class="fa fa-check"></i><b>3.4.1</b> Minima/maxima of PC1/2 with warp grids<span></span></a></li>
<li class="chapter" data-level="3.4.2" data-path="geometric-morphometrics-1.html"><a href="geometric-morphometrics-1.html#composite-pca-with-warp-grids"><i class="fa fa-check"></i><b>3.4.2</b> Composite PCA with warp grids<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="geometric-morphometrics-1.html"><a href="geometric-morphometrics-1.html#procrustes-anova-allometry"><i class="fa fa-check"></i><b>3.5</b> Procrustes ANOVA: Allometry<span></span></a></li>
<li class="chapter" data-level="3.6" data-path="geometric-morphometrics-1.html"><a href="geometric-morphometrics-1.html#procrustes-anova-shape-and-size"><i class="fa fa-check"></i><b>3.6</b> Procrustes ANOVA: Shape and size<span></span></a></li>
<li class="chapter" data-level="3.7" data-path="geometric-morphometrics-1.html"><a href="geometric-morphometrics-1.html#modularity"><i class="fa fa-check"></i><b>3.7</b> Modularity<span></span></a></li>
<li class="chapter" data-level="3.8" data-path="geometric-morphometrics-1.html"><a href="geometric-morphometrics-1.html#morphological-integration"><i class="fa fa-check"></i><b>3.8</b> Morphological integration<span></span></a></li>
<li class="chapter" data-level="3.9" data-path="geometric-morphometrics-1.html"><a href="geometric-morphometrics-1.html#trajectory-analysis"><i class="fa fa-check"></i><b>3.9</b> Trajectory analysis<span></span></a></li>
<li class="chapter" data-level="3.10" data-path="geometric-morphometrics-1.html"><a href="geometric-morphometrics-1.html#morphological-disparity"><i class="fa fa-check"></i><b>3.10</b> Morphological disparity<span></span></a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="gm-v-linear.html"><a href="gm-v-linear.html"><i class="fa fa-check"></i><b>4</b> GM v Linear<span></span></a>
<ul>
<li class="chapter" data-level="4.1" data-path="gm-v-linear.html"><a href="gm-v-linear.html#load-packages-data-2"><i class="fa fa-check"></i><b>4.1</b> Load packages + data<span></span></a></li>
<li class="chapter" data-level="4.2" data-path="gm-v-linear.html"><a href="gm-v-linear.html#generalised-procrustes-analysis-2"><i class="fa fa-check"></i><b>4.2</b> Generalised Procrustes Analysis<span></span></a></li>
<li class="chapter" data-level="4.3" data-path="gm-v-linear.html"><a href="gm-v-linear.html#shape-vs-linear-data-shape-vars"><i class="fa fa-check"></i><b>4.3</b> Shape vs linear data (shape vars)<span></span></a></li>
<li class="chapter" data-level="4.4" data-path="gm-v-linear.html"><a href="gm-v-linear.html#shape-vs-linear-data-linear-vars"><i class="fa fa-check"></i><b>4.4</b> Shape vs linear data (linear vars)<span></span></a></li>
</ul></li>
<li><a href="references.html#references">References<span></span></a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Morphologically similar, but regionally distinct: Perdiz arrow points from Caddo burial contexts in the American Southeast</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="header">
<h1 class="title">Morphologically similar, but regionally distinct: Perdiz arrow points from Caddo burial contexts in the American Southeast</h1>
<p class="author"><em>Robert Z. Selden, Jr.<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a></em></p>
<p class="date"><em>2022-04-06</em></p>
</div>
<div id="preface" class="section level1 unnumbered hasAnchor">
<h1>Preface<a href="index.html#preface" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<blockquote>
<p>If you describe something with great accuracy, you describe it as closely as you can to what it is, to its true value. If you describe something with great precision, you do so in the greatest possible detail, even though that detail may not necessarily be the true value of the thing being described. <strong>Simon Winchester, <em>The Perfectionists: How Precision Engineers Created the Modern World</em></strong></p>
</blockquote>
<p>Generally considered diagnostic of Late Prehistoric Toyah assemblages, Perdiz arrow points are characteristic of the transition from the Late Prehistoric to the Protohistoric. If larger Perdiz arrow points from Caddo burials are conceived of as products of trade and/or exchange with Toyah groups, then those with longer blade lengths provide inference to shifts in Caddo selective preference, while those with shorter blade lengths evince local approaches to resharpening and/or retouch that were uniquely Caddo. This study asks whether linear shape variables evince discrete regional resharpening strategies, whether morphological trajectories differ between the northern and southern behavioural regions, and whether morphological disparity differs between larger and smaller size classes, as defined by differences in blade length. Results demonstrate differing regional resharpening strategies, and distinct morphological trajectories for Perdiz arrow points included as Caddo mortuary offerings in the northern and southern behavioural regions. Perdiz arrow point shapes were not found to differ in the large size class between the northern and southern behavioural regions, demonstrating consistency in Caddo preference, or—alternatively—Toyah manufacture. However, differences in the small size class suggest distinct local approaches to resharpening and/or retouch by Caddo knappers. Caddo groups that occupied the southern behavioural region may have also been less selective, preferring Perdiz arrow points with a greater range of diversity in shape and size, while their counterparts to the north preferred a more standardized product.</p>
<div class="figure"><span style="display:block;" id="fig:hypothesis"></span>
<img src="img/img2/hypothesis.viz.png" alt="Visualization of expectations 1) for Perdiz arrow point trajectories, where (a) morphological disparity in shape (dashed lines) is greater at a larger size (selective preference), and/or (b) morphological disparity in shape is greater at a smaller size (resharpening and/or retouch); where 2) morphological disparity in Perdiz arrow points is expected to be greater where blade length is shorter, due to episodic reduction or retouch; and 3) illustration of Perdiz arrow points offerings included with Caddo burials. " width="100%" />
<p class="caption">
Figure 0.1: Visualization of expectations 1) for Perdiz arrow point trajectories, where (a) morphological disparity in shape (dashed lines) is greater at a larger size (selective preference), and/or (b) morphological disparity in shape is greater at a smaller size (resharpening and/or retouch); where 2) morphological disparity in Perdiz arrow points is expected to be greater where blade length is shorter, due to episodic reduction or retouch; and 3) illustration of Perdiz arrow points offerings included with Caddo burials.
</p>
</div>
<div id="primary-findings" class="section level2 hasAnchor" number="0.1">
<h2><span class="header-section-number">0.1</span> Primary findings<a href="index.html#primary-findings" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div class="figure"><span style="display:block;" id="fig:MAP"></span>
<img src="img/img2/sca.jpg" alt="Location of sites with Perdiz arrow points included in the study, extent of major drainage basins (Neches [brown], Sabine [maroon], and Red [navy] Rivers), and extent of the ancestral Caddo region (white). In the context of this effort, those sites from the Red River basin (navy) constitute the `northern behavioural region`, while those from the Sabine (maroon) and Neches (brown) River basins represent the `southern behavioural region`." width="100%" />
<p class="caption">
Figure 0.2: Location of sites with Perdiz arrow points included in the study, extent of major drainage basins (Neches [brown], Sabine [maroon], and Red [navy] Rivers), and extent of the ancestral Caddo region (white). In the context of this effort, those sites from the Red River basin (navy) constitute the <code>northern behavioural region</code>, while those from the Sabine (maroon) and Neches (brown) River basins represent the <code>southern behavioural region</code>.
</p>
</div>
<div id="linear-metrics" class="section level3 hasAnchor" number="0.1.1">
<h3><span class="header-section-number">0.1.1</span> Linear metrics<a href="index.html#linear-metrics" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>For this effort, the mean <code>maximum blade length</code> for Perdiz arrow points from each behavioural region was calculated and used to bin the sample into two groups (henceforth, <code>size classes</code>); one with a longer blade, and the other, shorter. Following the assignment of <code>size classes</code>, data were joined in advance of analysis.</p>
<p>The morphological form (shape + size) of the Perdiz arrow points was then split into <code>shape</code> (representing proportions between dimensions in an object) and <code>size</code> (sensu Klingenberg <span class="citation">(<a href="#ref-RN11105" role="doc-biblioref">2016</a>)</span>). <code>Size</code> and <code>shape</code> were calculated using the method of log-shape ratios proposed by Mosimann <span class="citation">(<a href="#ref-RN11104" role="doc-biblioref">1970</a>)</span>, where the <code>geometric mean</code> is considered a proxy for size, and shape was calculated as the log of isometric size subtracted from the log of each linear/dimensional measure.</p>
<p>An ANOVA was subsequently used to assess <code>shape</code> as a function of <code>size class</code>.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="index.html#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="do">## boxplots by size class</span></span>
<span id="cb1-2"><a href="index.html#cb1-2" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">&#39;img/boxplot-linear.png&#39;</span>)</span></code></pre></div>
<div class="figure"><span style="display:block;" id="fig:boxlg"></span>
<img src="img/boxplot-linear.png" alt="Boxplot by size class." width="100%" />
<p class="caption">
Figure 0.3: Boxplot by size class.
</p>
</div>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="index.html#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="do">## pca by size class</span></span>
<span id="cb2-2"><a href="index.html#cb2-2" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">&#39;img/pca-linear.png&#39;</span>)</span></code></pre></div>
<div class="figure"><span style="display:block;" id="fig:linpca"></span>
<img src="img/pca-linear.png" alt="PCA by size class." width="100%" />
<p class="caption">
Figure 0.4: PCA by size class.
</p>
</div>
<div id="results" class="section level4 hasAnchor" number="0.1.1.1">
<h4><span class="header-section-number">0.1.1.1</span> Results<a href="index.html#results" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li>Blade length <strong>differs significantly</strong> between large and small Perdiz arrow point <code>size classes</code> in the northern and southern behavioural regions</li>
<li>Shoulder width <strong>differs significantly</strong> between large and small Perdiz arrow point <code>size classes</code> in the southern behavioural region, but <strong>does not differ</strong> in the northern behavioural region</li>
<li>Width <strong>differs significantly</strong> between large and small Perdiz arrow point <code>size classes</code> in the southern behavioural region, but <strong>does not differ</strong> in the northern behavioural region</li>
<li>Thickness <strong>does not differ</strong> between large and small Perdiz arrow point <code>size classes</code> from either behavioural region</li>
<li>Stem length <strong>does not differ</strong> between large and small Perdiz arrow point <code>size classes</code> from either behavioural region</li>
<li>Stem width <strong>does not differ</strong> between large and small Perdiz arrow point <code>size classes</code> from either behavioural region</li>
</ul>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="index.html#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="do">## resharpening</span></span>
<span id="cb3-2"><a href="index.html#cb3-2" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">&#39;img/img2/linresharp.png&#39;</span>)</span></code></pre></div>
<div class="figure"><span style="display:block;" id="fig:linresharp"></span>
<img src="img/img2/linresharp.png" alt="Conceptual rendering of differences found to occur between Perdiz arrow point resharpening trajectories in the a, northern;  and b, southern behavioural regions. In both cases, attributes associated with hafting---`maximum stem length` and `stem width`---are morphologically stable. However, differential approaches to resharpening and/or retouch resulted in distinct shape differences in Perdiz arrow point blades. Resharpening and/or retouch efforts of Caddo knappers from the northern behavioural region was limited to `maximum blade length`, while those from the southern behavioural region were found to be more dynamic, including changes in `maximum blade length`, `width`, and `shoulder width` that occur between `size classes`." width="100%" />
<p class="caption">
Figure 0.5: Conceptual rendering of differences found to occur between Perdiz arrow point resharpening trajectories in the a, northern; and b, southern behavioural regions. In both cases, attributes associated with hafting—<code>maximum stem length</code> and <code>stem width</code>—are morphologically stable. However, differential approaches to resharpening and/or retouch resulted in distinct shape differences in Perdiz arrow point blades. Resharpening and/or retouch efforts of Caddo knappers from the northern behavioural region was limited to <code>maximum blade length</code>, while those from the southern behavioural region were found to be more dynamic, including changes in <code>maximum blade length</code>, <code>width</code>, and <code>shoulder width</code> that occur between <code>size classes</code>.
</p>
</div>
</div>
</div>
<div id="geometric-morphometrics" class="section level3 hasAnchor" number="0.1.2">
<h3><span class="header-section-number">0.1.2</span> Geometric morphometrics<a href="index.html#geometric-morphometrics" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>This analysis follows recent geometric morphometric analyses of Perdiz arrow points (<a href="https://github.com/seldenlab/perdiz3/blob/main/ms/perdiz3/perdiz3.pdf">Selden and Dockall, in review</a>) and Gahagan biface morphology across the southern Caddo area and central Texas <span class="citation">(<a href="#ref-RN11783" role="doc-biblioref">Selden Jr., Dockall, and Shafer 2018</a>; <a href="#ref-RN21001" role="doc-biblioref">Selden Jr., Dockall, and Dubied 2020</a>)</span>, where Perdiz arrow points and Gahagan bifaces have been found to differ in shape across the same geography as Caddo bottles <span class="citation">(<a href="#ref-RN11801" role="doc-biblioref">Selden Jr. 2018a</a>, <a href="#ref-RN11782" role="doc-biblioref">2018b</a>, <a href="#ref-RN11716" role="doc-biblioref">2019</a>, <a href="#ref-RN8312" role="doc-biblioref">2021</a>)</span>.</p>
<p>It also follows a recent geometric morphometric analysis in which Perdiz arrow points from the ancestral Caddo area were found to differ as a function of time, raw material, and burial context <span class="citation">(<a href="#ref-RN9551" role="doc-biblioref">Selden Jr et al. 2021</a>; <a href="#ref-RN8980" role="doc-biblioref">Selden Jr. et al. 2021</a>)</span>.</p>
<p>Landmark data were aligned to a global coordinate system <span class="citation">(<a href="#ref-RN11622" role="doc-biblioref">Kendall 1981</a>, <a href="#ref-RN11623" role="doc-biblioref">1984</a>; <a href="#ref-RN11563" role="doc-biblioref">Slice 2001</a>)</span>, achieved through generalized Procrustes superimposition <span class="citation">(<a href="#ref-RN478" role="doc-biblioref">Rohlf and Slice 1990</a>)</span> performed in R 4.1.3 <span class="citation">(<a href="#ref-R" role="doc-biblioref">R Core Development Team, 2022</a>)</span> using the <code>geomorph</code> library v. 4.0.3 <span class="citation">(<a href="#ref-RN11530" role="doc-biblioref">Adams et al. 2017</a>; <a href="#ref-RN1774" role="doc-biblioref">Adams and Otarola-Castillo 2013</a>; <a href="#ref-RN9565" role="doc-biblioref">Baken et al. 2021</a>)</span>. Procrustes superimposition translates, scales, and rotates the coordinate data to allow for comparisons among objects <span class="citation">(<a href="#ref-RN11564" role="doc-biblioref">Gower 1975</a>; <a href="#ref-RN478" role="doc-biblioref">Rohlf and Slice 1990</a>)</span>. The <code>geomorph</code> package uses a partial Procrustes superimposition that projects the aligned specimens into tangent space subsequent to alignment in preparation for the use of multivariate methods that assume linear space <span class="citation">(<a href="#ref-RN1646" role="doc-biblioref">Rohlf 1999</a>; <a href="#ref-RN11563" role="doc-biblioref">Slice 2001</a>)</span>.</p>
<p>Principal components analysis <span class="citation">(<a href="#ref-RN1746" role="doc-biblioref">Jolliffe 2002</a>)</span> was used to visualise shape variation among the Perdiz arrow points. The shape changes described by each principal axis are commonly visualized using thin-plate spline warping of a reference 3D mesh <span class="citation">(<a href="#ref-RN1731" role="doc-biblioref">Klingenberg 2013</a>; <a href="#ref-RN479" role="doc-biblioref">Sherratt et al. 2014</a>)</span>.</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="index.html#cb4-1" aria-hidden="true" tabindex="-1"></a><span class="do">## pca</span></span>
<span id="cb4-2"><a href="index.html#cb4-2" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">&#39;img/img2/gm-pca.png&#39;</span>)</span></code></pre></div>
<div class="figure"><span style="display:block;" id="fig:gmpca"></span>
<img src="img/img2/gm-pca.png" alt="PCA by size class, illustrating variability associated with basal morphology in PC1, and variability associated with blade morphology in PC2." width="100%" />
<p class="caption">
Figure 0.6: PCA by size class, illustrating variability associated with basal morphology in PC1, and variability associated with blade morphology in PC2.
</p>
</div>
<p>To assess whether shape changes with size (allometry) or differs by size class, Procrustes ANOVAs <span class="citation">(<a href="#ref-RN1749" role="doc-biblioref">Goodall 1991</a>)</span> were run that enlist effect-sizes (zscores) computed as standard deviates of the generated sampling distributions <span class="citation">(<a href="#ref-RN8446" role="doc-biblioref">Michael. L. Collyer, Sekora, and Adams 2015</a>)</span>. A residual randomization permutation procedure (RRPP; n = 10,000 permutations) was used for all Procrustes ANOVAs <span class="citation">(<a href="#ref-RN1655" role="doc-biblioref">Adams and Collyer 2015</a>; <a href="#ref-RN11775" role="doc-biblioref">M. L. Collyer and Adams 2018</a>)</span>, which has higher statistical power and a greater ability to identify patterns in the data should they be present <span class="citation">(<a href="#ref-RN1719" role="doc-biblioref">Anderson and Ter Braak 2003</a>)</span>. These results are detailed in Chapter 2.</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="index.html#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="do">## allometric (shape + size) trajectories by size class</span></span>
<span id="cb5-2"><a href="index.html#cb5-2" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">&#39;img/allometric.traj.png&#39;</span>)</span></code></pre></div>
<div class="figure"><span style="display:block;" id="fig:allometry"></span>
<img src="img/allometric.traj.png" alt="The homogeneity of slopes test indicates that the slopes associated with large and small Perdiz arrow points from the northern and southern behavioural regions differ. This figure plots the first principal component of _predicted_ values versus _size_ as a stylized graphic of the allometric trend. " width="100%" />
<p class="caption">
Figure 0.7: The homogeneity of slopes test indicates that the slopes associated with large and small Perdiz arrow points from the northern and southern behavioural regions differ. This figure plots the first principal component of <em>predicted</em> values versus <em>size</em> as a stylized graphic of the allometric trend.
</p>
</div>
<p>A phenotypic trajectory analysis <span class="citation">(<a href="#ref-RN8344" role="doc-biblioref">Adams and Collyer 2007</a>, <a href="#ref-RN8550" role="doc-biblioref">2009</a>; <a href="#ref-RN8569" role="doc-biblioref">M. L. Collyer and Adams 2007</a>, <a href="#ref-RN8533" role="doc-biblioref">2013</a>; <a href="#ref-RN8446" role="doc-biblioref">Michael. L. Collyer, Sekora, and Adams 2015</a>)</span> was used to assess whether Caddo reduction or retouch trajectories differ between the northern and southern behavioural regions. The trajectory analysis was followed by an analysis of morphological disparity <span class="citation">(<a href="#ref-RN11107" role="doc-biblioref">M. L. Collyer and Adams 2020</a>; <a href="#ref-RN7041" role="doc-biblioref">Foote 1993</a>; <a href="#ref-RN5694" role="doc-biblioref">Zelditch et al. 2004</a>)</span> to identify potential differences in morphological diversity between size classes.</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="index.html#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Perdiz arrow point shape trajectories by size class</span></span>
<span id="cb6-2"><a href="index.html#cb6-2" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">&#39;img/img2/trajectory.png&#39;</span>)</span></code></pre></div>
<div class="figure"><span style="display:block;" id="fig:trajectory"></span>
<img src="img/img2/trajectory.png" alt="Visualization of two shape trajectories for Perdiz arrow points (large/small) that occur in the northern and southern behavioural regions. Shape trajectories are projected onto the first two principal components of between-group shape variation (based on covariance matrix of group means). Trajectories are shown as lines, where the northern behavioural region is represented by squares, and the southern behavioural region by triangles. Transformations are added to facilitate an understanding of shape differences corresponding to large and small Perdiz arrow point shapes from each behavioural region." width="100%" />
<p class="caption">
Figure 0.8: Visualization of two shape trajectories for Perdiz arrow points (large/small) that occur in the northern and southern behavioural regions. Shape trajectories are projected onto the first two principal components of between-group shape variation (based on covariance matrix of group means). Trajectories are shown as lines, where the northern behavioural region is represented by squares, and the southern behavioural region by triangles. Transformations are added to facilitate an understanding of shape differences corresponding to large and small Perdiz arrow point shapes from each behavioural region.
</p>
</div>
<div id="results-1" class="section level4 hasAnchor" number="0.1.2.1">
<h4><span class="header-section-number">0.1.2.1</span> Results<a href="index.html#results-1" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li>Allometry is <strong>significant</strong>, and the homogeneity of slopes test indicates that the slopes associated with large and small Perdiz arrow points from the northern and southern behavioural regions differ</li>
<li>Modularity is <strong>significant</strong></li>
<li>Morphological integration is <strong>significant</strong></li>
<li>Results of the trajectory analysis demonstrate a <strong>significant difference</strong> between shape trajectories in the northern and southern behavioural regions</li>
<li>Results of the morphological disparity analysis by <em>shape</em> demonstrate that the large Perdiz arrow points from the southern behavioural region occupy a <strong>significantly greater</strong> range of morphospace than the large and small Perdiz arrow points from the northern behavioural region</li>
<li>Results of the morphological disparity analysis by <em>size</em> demonstrate that the large Perdiz arrow points from the southern behavioural region occupy a <strong>significantly greater</strong> range of morphospace than the large Perdiz arrow points in the northern behavioural region</li>
</ul>
</div>
</div>
</div>
<div id="acknowledgments" class="section level2 hasAnchor" number="0.2">
<h2><span class="header-section-number">0.2</span> Acknowledgments<a href="index.html#acknowledgments" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>I extend my gratitude to the Caddo Nation of Oklahoma, the Caddo Nation Tribal Council, Tribal Chairman, and Tribal Historic Preservation Office for their guidance related to the development of the scanning protocols, for permission and access to NAGPRA and previously repatriated collections, and for frank discussions related to language surrounding burial contexts associated with Caddo children. Thanks also to the Anthropology and Archaeology Laboratory at Stephen F. Austin State University for the requisite permissions and access to the NAGPRA items from the Washington Square Mound site and Turner collection, and to Tom A. Middlebrook for brokering access to the Perdiz arrow points from Caddo burials at the Morse Mound site. Thanks also to Michael J. Shott, Christian S. Hoggard, Emma Sherratt, Dean C. Adams, Michael L. Collyer, Lauren Butaric, David K. Thulman, Bonnie Etter, and Kersten Bergstrom for their constructive criticisms, comments, and suggestions throughout the development of this research program, and to the editors and anonymous reviewers for their comments and constructive criticisms, which further improved the manuscript and supplementary materials.</p>
</div>
<div id="funding" class="section level2 hasAnchor" number="0.3">
<h2><span class="header-section-number">0.3</span> Funding<a href="index.html#funding" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Components of the analytical workflow were developed and funded by a Preservation Technology and Training grant (P14AP00138) to RZS from the National Center for Preservation Technology and Training, as well as grants to RZS from the Caddo Nation of Oklahoma, National Forests and Grasslands in Texas (15-PA-11081300-033) and the United States Forest Service (20-PA-11081300-074). Additional support for this project was provided by the Heritage Research Center at Stephen F. Austin State University.</p>
</div>
<div id="data-management" class="section level2 hasAnchor" number="0.4">
<h2><span class="header-section-number">0.4</span> Data management<a href="index.html#data-management" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>The data and analysis code associated with this project can be accessed through this document or the <a href="https://seldenlab.github.io/perdiz.4/">GitHub</a> repository, and is digitally curated on the Open Science Framework (<a href="https://osf.io/85bu7/">DOI 10.17605/OSF.IO/85BU7</a>). The replicable nature of this undertaking provides a means for others to critically assess and evaluate the various analytical components of this study <span class="citation">(<a href="#ref-RN20915" role="doc-biblioref">Gray and Marwick 2019</a>; <a href="#ref-RN20916" role="doc-biblioref">Peng 2011</a>; <a href="#ref-RN20917" role="doc-biblioref">Gandrud 2014</a>)</span>, which is a necessary requirement for the production of reliable knowledge.</p>
<p>Reproducibility projects in <a href="https://osf.io/ezcuj/">psychology</a> and <a href="https://www.cos.io/rpcb">cancer biology</a> are impacting current research practices across all domains. Examples of reproducible research are becoming more abundant in archaeology <span class="citation">(<a href="#ref-RN20804" role="doc-biblioref">Marwick 2016</a>; <a href="#ref-RN21009" role="doc-biblioref">Ivanovaite et al. 2020</a>; <a href="#ref-RN21001" role="doc-biblioref">Selden Jr., Dockall, and Dubied 2020</a>)</span>, and the next generation of archaeologists are learning those tools and methods needed to reproduce and/or replicate research results <span class="citation">(<a href="#ref-RN21007" role="doc-biblioref">Marwick et al. 2019</a>)</span>. Reproducible and replicable research work flows are often employed at the highest levels of humanities-based inquiries to mitigate concern or doubt regarding proper execution, and is of particular import should the results have—explicitly or implicitly—a major impact on scientific progress <span class="citation">(<a href="#ref-RN21008" role="doc-biblioref">Peels and Bouter 2018</a>)</span>.</p>
</div>
<div id="colophon" class="section level2 hasAnchor" number="0.5">
<h2><span class="header-section-number">0.5</span> Colophon<a href="index.html#colophon" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>This version of the analysis was generated on 2022-04-06 12:52:52 using the following computational environment and dependencies:</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="index.html#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="co"># what R packages and versions were used?</span></span>
<span id="cb7-2"><a href="index.html#cb7-2" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> (<span class="st">&quot;devtools&quot;</span> <span class="sc">%in%</span> <span class="fu">installed.packages</span>()) devtools<span class="sc">::</span><span class="fu">session_info</span>()</span></code></pre></div>
<pre><code>## - Session info ------------------------------------------------------------------------------------
##  setting  value
##  version  R version 4.1.3 (2022-03-10)
##  os       Windows 10 x64 (build 19044)
##  system   x86_64, mingw32
##  ui       RStudio
##  language (EN)
##  collate  English_United States.1252
##  ctype    English_United States.1252
##  tz       America/Chicago
##  date     2022-04-06
##  rstudio  2021.09.0+351 Ghost Orchid (desktop)
##  pandoc   2.14.0.3 @ C:/Program Files/RStudio/bin/pandoc/ (via rmarkdown)
## 
## - Packages ----------------------------------------------------------------------------------------
##  package       * version    date (UTC) lib source
##  abind           1.4-5      2016-07-21 [1] CRAN (R 4.1.0)
##  ape             5.6-2      2022-03-02 [1] CRAN (R 4.1.3)
##  assertthat      0.2.1      2019-03-21 [1] CRAN (R 4.1.0)
##  backports       1.4.1      2021-12-13 [1] CRAN (R 4.1.2)
##  bezier          1.1.2      2018-12-14 [1] CRAN (R 4.1.0)
##  bookdown        0.25       2022-03-16 [1] CRAN (R 4.1.3)
##  brew            1.0-7      2022-02-04 [1] CRAN (R 4.1.2)
##  brio            1.1.3      2021-11-30 [1] CRAN (R 4.1.2)
##  broom           0.7.12     2022-01-28 [1] CRAN (R 4.1.2)
##  bslib           0.3.1      2021-10-06 [1] CRAN (R 4.1.1)
##  cachem          1.0.6      2021-08-19 [1] CRAN (R 4.1.1)
##  callr           3.7.0      2021-04-20 [1] CRAN (R 4.1.0)
##  car             3.0-12     2021-11-06 [1] CRAN (R 4.1.1)
##  carData         3.0-5      2022-01-06 [1] CRAN (R 4.1.2)
##  cellranger      1.1.0      2016-07-27 [1] CRAN (R 4.1.0)
##  cli             3.2.0      2022-02-14 [1] CRAN (R 4.1.2)
##  cluster       * 2.1.2      2021-04-17 [2] CRAN (R 4.1.3)
##  colorspace      2.0-3      2022-02-21 [1] CRAN (R 4.1.3)
##  cowplot         1.1.1      2020-12-30 [1] CRAN (R 4.1.0)
##  crayon          1.5.1      2022-03-26 [1] CRAN (R 4.1.3)
##  data.table      1.14.2     2021-09-27 [1] CRAN (R 4.1.1)
##  DBI             1.1.2      2021-12-20 [1] CRAN (R 4.1.2)
##  dbplyr          2.1.1      2021-04-06 [1] CRAN (R 4.1.0)
##  desc            1.4.1      2022-03-06 [1] CRAN (R 4.1.3)
##  devtools        2.4.3      2021-11-30 [1] CRAN (R 4.1.2)
##  digest          0.6.29     2021-12-01 [1] CRAN (R 4.1.2)
##  dplyr         * 1.0.8      2022-02-08 [1] CRAN (R 4.1.2)
##  dtplyr          1.2.1      2022-01-19 [1] CRAN (R 4.1.2)
##  ellipsis        0.3.2      2021-04-29 [1] CRAN (R 4.1.0)
##  EnvStats      * 2.7.0      2022-03-07 [1] CRAN (R 4.1.3)
##  evaluate        0.15       2022-02-18 [1] CRAN (R 4.1.3)
##  fansi           1.0.3      2022-03-24 [1] CRAN (R 4.1.3)
##  farver          2.1.0      2021-02-28 [1] CRAN (R 4.1.0)
##  fastmap         1.1.0      2021-01-25 [1] CRAN (R 4.1.0)
##  forcats       * 0.5.1      2021-01-27 [1] CRAN (R 4.1.0)
##  fs              1.5.2      2021-12-08 [1] CRAN (R 4.1.2)
##  gargle          1.2.0      2021-07-02 [1] CRAN (R 4.1.0)
##  generics        0.1.2      2022-01-31 [1] CRAN (R 4.1.2)
##  geomorph      * 4.0.3.999  2022-04-04 [1] Github (geomorphR/geomorph@33648a3)
##  ggExtra       * 0.10.0     2022-03-23 [1] CRAN (R 4.1.3)
##  ggfortify     * 0.4.14     2022-01-05 [1] Github (sinhrks/ggfortify@619d40e)
##  ggplot2       * 3.3.5      2021-06-25 [1] CRAN (R 4.1.0)
##  ggpubr        * 0.4.0.999  2022-01-05 [1] Github (kassambara/ggpubr@ac5a01f)
##  ggsignif        0.6.3      2021-09-09 [1] CRAN (R 4.1.1)
##  git2r           0.30.1     2022-03-16 [1] CRAN (R 4.1.3)
##  glue            1.6.2      2022-02-24 [1] CRAN (R 4.1.3)
##  googledrive     2.0.0      2021-07-08 [1] CRAN (R 4.1.0)
##  googlesheets4   1.0.0      2021-07-21 [1] CRAN (R 4.1.0)
##  gridExtra       2.3        2017-09-09 [1] CRAN (R 4.1.0)
##  gtable          0.3.0      2019-03-25 [1] CRAN (R 4.1.0)
##  haven           2.4.3      2021-08-04 [1] CRAN (R 4.1.0)
##  here          * 1.0.1.9000 2021-11-08 [1] Github (r-lib/here@b66d1ab)
##  highr           0.9        2021-04-16 [1] CRAN (R 4.1.0)
##  hms             1.1.1      2021-09-26 [1] CRAN (R 4.1.1)
##  htmltools       0.5.2      2021-08-25 [1] CRAN (R 4.1.1)
##  htmlwidgets     1.5.4      2021-09-08 [1] CRAN (R 4.1.1)
##  httpuv          1.6.5      2022-01-05 [1] CRAN (R 4.1.2)
##  httr            1.4.2      2020-07-20 [1] CRAN (R 4.1.0)
##  jpeg            0.1-9      2021-07-24 [1] CRAN (R 4.1.0)
##  jquerylib       0.1.4      2021-04-26 [1] CRAN (R 4.1.0)
##  jsonlite        1.8.0      2022-02-22 [1] CRAN (R 4.1.3)
##  knitr           1.38       2022-03-25 [1] CRAN (R 4.1.3)
##  labeling        0.4.2      2020-10-20 [1] CRAN (R 4.1.0)
##  later           1.3.0      2021-08-18 [1] CRAN (R 4.1.1)
##  lattice         0.20-45    2021-09-22 [2] CRAN (R 4.1.3)
##  lifecycle       1.0.1      2021-09-24 [1] CRAN (R 4.1.1)
##  lubridate       1.8.0      2021-10-07 [1] CRAN (R 4.1.1)
##  magrittr        2.0.3      2022-03-30 [1] CRAN (R 4.1.3)
##  MASS            7.3-55     2022-01-16 [2] CRAN (R 4.1.3)
##  Matrix        * 1.4-0      2021-12-08 [2] CRAN (R 4.1.3)
##  memoise         2.0.1      2021-11-26 [1] CRAN (R 4.1.2)
##  mime            0.12       2021-09-28 [1] CRAN (R 4.1.1)
##  miniUI          0.1.1.1    2018-05-18 [1] CRAN (R 4.1.0)
##  modelr          0.1.8      2020-05-19 [1] CRAN (R 4.1.0)
##  munsell         0.5.0      2018-06-12 [1] CRAN (R 4.1.0)
##  nlme            3.1-155    2022-01-16 [2] CRAN (R 4.1.3)
##  pillar          1.7.0      2022-02-01 [1] CRAN (R 4.1.2)
##  pkgbuild        1.3.1      2021-12-20 [1] CRAN (R 4.1.2)
##  pkgconfig       2.0.3      2019-09-22 [1] CRAN (R 4.1.0)
##  pkgload         1.2.4      2021-11-30 [1] CRAN (R 4.1.2)
##  png             0.1-7      2013-12-03 [1] CRAN (R 4.1.0)
##  prettyunits     1.1.1      2020-01-24 [1] CRAN (R 4.1.0)
##  processx        3.5.3      2022-03-25 [1] CRAN (R 4.1.3)
##  promises        1.2.0.1    2021-02-11 [1] CRAN (R 4.1.0)
##  ps              1.6.0      2021-02-28 [1] CRAN (R 4.1.0)
##  purrr         * 0.3.4      2020-04-17 [1] CRAN (R 4.1.0)
##  R6              2.5.1      2021-08-19 [1] CRAN (R 4.1.1)
##  Rcpp            1.0.8.3    2022-03-17 [1] CRAN (R 4.1.3)
##  readr         * 2.1.2      2022-01-30 [1] CRAN (R 4.1.2)
##  readxl          1.4.0      2022-03-28 [1] CRAN (R 4.1.3)
##  remotes         2.4.2      2021-11-30 [1] CRAN (R 4.1.2)
##  reprex          2.0.1      2021-08-05 [1] CRAN (R 4.1.0)
##  rgl           * 0.108.3    2021-11-21 [1] CRAN (R 4.1.2)
##  rjson           0.2.21     2022-01-09 [1] CRAN (R 4.1.2)
##  rlang           1.0.2      2022-03-04 [1] CRAN (R 4.1.3)
##  rmarkdown       2.13       2022-03-10 [1] CRAN (R 4.1.3)
##  Rook            1.1-1      2014-10-20 [1] CRAN (R 4.1.0)
##  rprojroot       2.0.3      2022-04-02 [1] CRAN (R 4.1.3)
##  RRPP          * 1.2.3      2022-03-18 [1] Github (mlcollyer/RRPP@6e6eb22)
##  rstatix         0.7.0      2021-02-13 [1] CRAN (R 4.1.0)
##  rstudioapi      0.13       2020-11-12 [1] CRAN (R 4.1.0)
##  rvest           1.0.2      2021-10-16 [1] CRAN (R 4.1.1)
##  sass            0.4.1      2022-03-23 [1] CRAN (R 4.1.3)
##  scales          1.1.1      2020-05-11 [1] CRAN (R 4.1.0)
##  sessioninfo     1.2.2      2021-12-06 [1] CRAN (R 4.1.2)
##  shiny           1.7.1      2021-10-02 [1] CRAN (R 4.1.1)
##  StereoMorph   * 1.6.4      2021-05-25 [1] CRAN (R 4.1.0)
##  stringi         1.7.6      2021-11-29 [1] CRAN (R 4.1.1)
##  stringr       * 1.4.0      2019-02-10 [1] CRAN (R 4.1.0)
##  svgViewR        1.4.1      2020-10-03 [1] CRAN (R 4.1.0)
##  testthat        3.1.3      2022-03-29 [1] CRAN (R 4.1.3)
##  tibble        * 3.1.6      2021-11-07 [1] CRAN (R 4.1.1)
##  tidyr         * 1.2.0      2022-02-01 [1] CRAN (R 4.1.2)
##  tidyselect      1.1.2      2022-02-21 [1] CRAN (R 4.1.3)
##  tidyverse     * 1.3.1.9000 2022-01-05 [1] Github (tidyverse/tidyverse@6186fbf)
##  tiff            0.1-11     2022-01-31 [1] CRAN (R 4.1.2)
##  tzdb            0.3.0      2022-03-28 [1] CRAN (R 4.1.3)
##  usethis         2.1.5      2021-12-09 [1] CRAN (R 4.1.2)
##  utf8            1.2.2      2021-07-24 [1] CRAN (R 4.1.0)
##  vctrs           0.4.0      2022-03-30 [1] CRAN (R 4.1.3)
##  wesanderson   * 0.3.6.9000 2021-11-08 [1] Github (karthik/wesanderson@651c944)
##  withr           2.5.0      2022-03-03 [1] CRAN (R 4.1.3)
##  xfun            0.30       2022-03-02 [1] CRAN (R 4.1.3)
##  xml2            1.3.3      2021-11-30 [1] CRAN (R 4.1.2)
##  xtable          1.8-4      2019-04-21 [1] CRAN (R 4.1.0)
##  yaml            2.3.5      2022-02-21 [1] CRAN (R 4.1.2)
## 
##  [1] C:/Users/seldenjrz/Documents/R/win-library/4.1
##  [2] C:/Program Files/R/R-4.1.3/library
## 
## ---------------------------------------------------------------------------------------------------</code></pre>
<p>Current Git commit details are:</p>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="index.html#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="co"># where can I find this commit? </span></span>
<span id="cb9-2"><a href="index.html#cb9-2" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> (<span class="st">&quot;git2r&quot;</span> <span class="sc">%in%</span> <span class="fu">installed.packages</span>() <span class="sc">&amp;</span> git2r<span class="sc">::</span><span class="fu">in_repository</span>(<span class="at">path =</span> <span class="st">&quot;.&quot;</span>)) git2r<span class="sc">::</span><span class="fu">repository</span>(here<span class="sc">::</span><span class="fu">here</span>())  </span></code></pre></div>
<pre><code>## Local:    main E:/github/perdiz.4
## Remote:   main @ origin (https://github.com/seldenlab/perdiz.4)
## Head:     [738240e] 2022-04-06: &lt;edit ms&gt;</code></pre>

</div>
</div>
<h3>References<a href="references.html#references" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-RN8344" class="csl-entry">
Adams, Dean C., and Michael L. Collyer. 2007. <span>“Analysis of Character Divergence Along Environmental Gradients and Other Covariates.”</span> Journal Article. <em>Evolution</em> 61 (3): 510–15. <a href="https://doi.org/10.1111/j.1558-5646.2007.00063.x">https://doi.org/10.1111/j.1558-5646.2007.00063.x</a>.
</div>
<div id="ref-RN8550" class="csl-entry">
———. 2009. <span>“A General Framework for the Analysis of Phenotypic Trajectories in Evolutionary Studies.”</span> Journal Article. <em>Evolution</em> 63 (5): 1143–54. <a href="https://doi.org/10.1111/j.1558-5646.2009.00649.x">https://doi.org/10.1111/j.1558-5646.2009.00649.x</a>.
</div>
<div id="ref-RN1655" class="csl-entry">
———. 2015. <span>“<span class="nocase">Permutation Tests for Phylogenetic Comparative Analyses of High-Dimensional Shape Data: What you Shuffle Matters</span>.”</span> <em>Evolution</em> 69 (3): 823–29. <a href="https://doi.org/10.1111/evo.12596">https://doi.org/10.1111/evo.12596</a>.
</div>
<div id="ref-RN11530" class="csl-entry">
Adams, Dean C., Michael L. Collyer, Antigoni Kaliontzopoulou, and Emma Sherratt. 2017. <span>“<span class="nocase">Package geomorph: Geometric Morphometric Analyses of Landmark Data. R package version 3.0.5</span>.”</span> <a href="http://geomorphr.github.io/geomorph/">http://geomorphr.github.io/geomorph/</a>.
</div>
<div id="ref-RN1774" class="csl-entry">
Adams, Dean C., and Erik Otarola-Castillo. 2013. <span>“<span class="nocase">geomorph: An R Package for the Collection and Analysis of Geometric Morphometric Shape Data</span>.”</span> <em>Methods in Ecology and Evolution</em> 4 (4): 393–99. <a href="https://doi.org/10.1111/2041-210x.12035">https://doi.org/10.1111/2041-210x.12035</a>.
</div>
<div id="ref-RN1719" class="csl-entry">
Anderson, M. J., and C. J. F. Ter Braak. 2003. <span>“<span class="nocase">Permutation Tests for Multi-Factoral Analysis of Variance</span>.”</span> <em>Journal of Statistical Computation and Simulation</em> 73 (2): 85–113. <a href="https://doi.org/10.1080=0094965021000015558">https://doi.org/10.1080=0094965021000015558</a>.
</div>
<div id="ref-RN9565" class="csl-entry">
Baken, Erica K., Michael L. Collyer, Antigoni Kaliontzopoulou, and Dean C. Adams. 2021. <span>“<span class="nocase">geomorph v4.0 and gmShiny: Enhanced analytics and a new graphical interface for a comprehensive morphometric experience</span>.”</span> Journal Article. <em>Methods in Ecology and Evolution</em>. <a href="https://doi.org/10.1111/2041-210x.13723">https://doi.org/10.1111/2041-210x.13723</a>.
</div>
<div id="ref-RN8569" class="csl-entry">
Collyer, Michael L., and Dean C. Adams. 2007. <span>“Analysis of Two-State Multivariate Phenotypic Change in Ecological Studies.”</span> Journal Article. <em>Ecology</em> 88 (3): 683–92. <a href="https://doi.org/10.1890/06-0727">https://doi.org/10.1890/06-0727</a>.
</div>
<div id="ref-RN8533" class="csl-entry">
———. 2013. <span>“Phenotypic Trajectory Analysis: Comparison of Shape Change Patterns in Evolution and Ecology.”</span> Journal Article. <em>Hystrix</em> 24 (1): 75–83. <a href="https://doi.org/doi:10.4404/hystrix-24.1-6298">https://doi.org/doi:10.4404/hystrix-24.1-6298</a>.
</div>
<div id="ref-RN11775" class="csl-entry">
———. 2018. <span>“<span class="nocase">RRPP: An R Package for Fitting Linear Models to High-Dimensional Data using Residual Randomization</span>.”</span> <em>Methods in Ecology and Evolution</em> 9 (7): 1772–79. https://doi.org/<a href="https://doi.org/10.1111/2041-210X.13029">https://doi.org/10.1111/2041-210X.13029</a>.
</div>
<div id="ref-RN11107" class="csl-entry">
———. 2020. <span>“Phylogenetically Aligned Component Analysis.”</span> Journal Article. <em>Methods in Ecology and Evolution</em> 12 (2): 359–72. <a href="https://doi.org/10.1111/2041-210x.13515">https://doi.org/10.1111/2041-210x.13515</a>.
</div>
<div id="ref-RN8446" class="csl-entry">
Collyer, Michael. L., David. J. Sekora, and Dean. C. Adams. 2015. <span>“A Method for Analysis of Phenotypic Change for Phenotypes Described by High-Dimensional Data.”</span> Journal Article. <em>Heredity (Edinb)</em> 115 (4): 357–65. <a href="https://doi.org/10.1038/hdy.2014.75">https://doi.org/10.1038/hdy.2014.75</a>.
</div>
<div id="ref-RN7041" class="csl-entry">
Foote, M. 1993. <span>“Contributions of Individual Taxa to Overall Morphological Disparity.”</span> Journal Article. <em>Paleobiology</em> 19 (4): 403–19.
</div>
<div id="ref-RN20917" class="csl-entry">
Gandrud, Christopher. 2014. <em>Reproducible Research with r and RStudio</em>. The r Series. London: CRC Press.
</div>
<div id="ref-RN1749" class="csl-entry">
Goodall, Colin. 1991. <span>“<span class="nocase">Procrustes Methods in the Statistical Analysis of Shape</span>.”</span> <em>Journal of the Royal Statistical Society. Series B (Methodological)</em> 53 (2): 285–339.
</div>
<div id="ref-RN11564" class="csl-entry">
Gower, J. C. 1975. <span>“<span>Generalized Procrustes Analysis</span>.”</span> <em>Psychometrika</em> 40 (1): 33–51. https://doi.org/<a href="https://doi.org/10.1007/BF02291478">https://doi.org/10.1007/BF02291478</a>.
</div>
<div id="ref-RN20915" class="csl-entry">
Gray, Charles T., and Ben Marwick. 2019. <span>“Truth, Proof, and Reproducibility: There’s No Counter-Attack for the Codeless.”</span> In <em>Statistics and Data Science</em>, 111–29. Communications in Computer and Information Science. <a href="https://doi.org/10.1007/978-981-15-1960-4_8">https://doi.org/10.1007/978-981-15-1960-4_8</a>.
</div>
<div id="ref-RN21009" class="csl-entry">
Ivanovaite, Livija, Kamil Serwatka, Christian Steven Hoggard, Florian Sauer, and Felix Riede. 2020. <span>“All These Fantastic Cultures? Research History and Regionalization in the Late Palaeolithic Tanged Point Cultures of Eastern Europe.”</span> <em>European Journal of Archaeology</em> 23 (2): 162–85. <a href="https://doi.org/10.1017/eaa.2019.59">https://doi.org/10.1017/eaa.2019.59</a>.
</div>
<div id="ref-RN1746" class="csl-entry">
Jolliffe, Ian T. 2002. <em>Principal Component Analysis</em>. New York: Springer.
</div>
<div id="ref-RN11622" class="csl-entry">
Kendall, David G. 1981. <span>“<span class="nocase">The Statistics of Shape</span>.”</span> In <em>Interpreting Multivariate Data</em>, edited by V. Barnett, 75–80. New York: Wiley.
</div>
<div id="ref-RN11623" class="csl-entry">
———. 1984. <span>“<span class="nocase">Shape Manifolds, Procrustean Metrics, and Complex Projective Spaces</span>.”</span> <em>Bulletin of the London Mathematical Society</em> 16 (2): 81–121. <a href="https://doi.org/10.1112/blms/16.2.81">https://doi.org/10.1112/blms/16.2.81</a>.
</div>
<div id="ref-RN1731" class="csl-entry">
Klingenberg, Christian Peter. 2013. <span>“<span class="nocase">Visualizations in Geometric Morphometrics: How to Read and How to Make Graphs Showing Shape Changes</span>.”</span> <em>Hystrix</em> 24 (1): 15–24. https://doi.org/<a href="http://dx.doi.org/10.4404/hystrix-24.1-7691">http://dx.doi.org/10.4404/hystrix-24.1-7691</a>.
</div>
<div id="ref-RN11105" class="csl-entry">
———. 2016. <span>“Size, Shape, and Form: Concepts of Allometry in Geometric Morphometrics.”</span> Journal Article. <em>Dev Genes Evol</em> 226 (3): 113–37. <a href="https://doi.org/10.1007/s00427-016-0539-2">https://doi.org/10.1007/s00427-016-0539-2</a>.
</div>
<div id="ref-RN20804" class="csl-entry">
Marwick, Ben. 2016. <span>“Computational Reproducibility in Archaeological Research: Basic Principles and a Case Study of Their Implementation.”</span> <em>Journal of Archaeological Method and Theory</em> 24 (2): 424–50. <a href="https://doi.org/10.1007/s10816-015-9272-9">https://doi.org/10.1007/s10816-015-9272-9</a>.
</div>
<div id="ref-RN21007" class="csl-entry">
Marwick, Ben, Li-Ying Wang, Ryan Robinson, and Hope Loiselle. 2019. <span>“How to Use Replication Assignments for Teaching Integrity in Empirical Archaeology.”</span> <em>Advances in Archaeological Practice</em> 8 (1): 78–86. <a href="https://doi.org/10.1017/aap.2019.38">https://doi.org/10.1017/aap.2019.38</a>.
</div>
<div id="ref-RN11104" class="csl-entry">
Mosimann, James E. 1970. <span>“Size Allometry: Size and Shape Variables with Characterizations of the Lognormal and Generalized Gamma Distributions.”</span> Journal Article. <em>Journal of the American Statistical Association</em> 65 (330): 930–45. <a href="https://doi.org/10.1080/01621459.1970.10481136">https://doi.org/10.1080/01621459.1970.10481136</a>.
</div>
<div id="ref-RN21008" class="csl-entry">
Peels, Rik, and Lex Bouter. 2018. <span>“Humanities Need a Replication Drive Too.”</span> <em>Nature</em> 558 (7710): 372. <a href="https://doi.org/10.1038/d41586-018-05454-w">https://doi.org/10.1038/d41586-018-05454-w</a>.
</div>
<div id="ref-RN20916" class="csl-entry">
Peng, Roger D. 2011. <span>“Reproducible Research in Computational Science.”</span> <em>Science</em> 334 (6060): 1226–27. <a href="https://doi.org/10.1126/science.1213847">https://doi.org/10.1126/science.1213847</a>.
</div>
<div id="ref-R" class="csl-entry">
R Core Development Team,. 2022. <em>R: A Language and Environment for Statistical Computing</em>. Vienna, Austria: R Foundation for Statistical Computing. <a href="http://www.R-project.org/">http://www.R-project.org/</a>.
</div>
<div id="ref-RN1646" class="csl-entry">
Rohlf, F. James. 1999. <span>“<span class="nocase">Shape Statistics: Procrustes Superimpositions and Tangent Spaces</span>.”</span> <em>Journal of Classification</em> 16 (2): 197–223. <a href="https://doi.org/10.1007/s003579900054">https://doi.org/10.1007/s003579900054</a>.
</div>
<div id="ref-RN478" class="csl-entry">
Rohlf, F. James, and Dennis Slice. 1990. <span>“<span class="nocase">Extensions of the Procrustes Method for the Optimal Superimposition of Landmarks</span>.”</span> <em>Systematic Zoology</em> 39 (1): 40–59. <a href="https://doi.org/10.2307/2992207">https://doi.org/10.2307/2992207</a>.
</div>
<div id="ref-RN9551" class="csl-entry">
Selden Jr, Robert Z., John E. Dockall, C. Britt Bousman, and Timothy K. Perttula. 2021. <span>“<span class="nocase">Shape as a function of time + raw material + burial context? An exploratory analysis of Perdiz arrow points from the ancestral Caddo area of the American Southeast</span>.”</span> Journal Article. <em>Journal of Archaeological Science: Reports</em> 37: 102916. <a href="https://doi.org/10.1016/j.jasrep.2021.102916">https://doi.org/10.1016/j.jasrep.2021.102916</a>.
</div>
<div id="ref-RN11801" class="csl-entry">
Selden Jr., Robert Z. 2018a. <span>“A Preliminary Study of Smithport Plain Bottle Morphology in the Southern Caddo Area.”</span> <em>Bulletin of the Texas Archeological Society</em> 89: 63–89.
</div>
<div id="ref-RN11782" class="csl-entry">
———. 2018b. <span>“<span class="nocase">Ceramic Morphological Organisation in the Southern Caddo Area: Quiddity of Shape for Hickory Engraved Bottles</span>.”</span> <em>Journal of Archaeological Science: Reports</em> 21: 884–96. <a href="https://doi.org/10.1016/j.jasrep.2018.08.045">https://doi.org/10.1016/j.jasrep.2018.08.045</a>.
</div>
<div id="ref-RN11716" class="csl-entry">
———. 2019. <span>“<span class="nocase">Ceramic Morphological Organisation in the Southern Caddo Area: The Clarence H. Webb Collections</span>.”</span> <em>Journal of Cultural Heritage</em> 35: 41–55. https://doi.org/<a href="https://doi.org/10.1016/j.culher.2018.07.002">https://doi.org/10.1016/j.culher.2018.07.002</a>.
</div>
<div id="ref-RN8312" class="csl-entry">
———. 2021. <span>“Louisiana Limitrophe: An Iterative Morphological Exegesis of Caddo Bottle and Biface Production.”</span> Book Section. In <em>Ancestral Caddo Ceramic Traditions</em>, edited by Duncan P. McKinnon, Jeffrey S. Girard, and Timothy K. Perttula, 258–76. Baton Rouge: LSU Press.
</div>
<div id="ref-RN8980" class="csl-entry">
Selden Jr., Robert Z., John E. Dockall, C. Britt Bousman, and Timothy K. Perttula. 2021. <span>“<span class="nocase">Supplementary materials for paper: Shape as a function of time + raw material + burial context? A preliminary analysis of Perdiz arrow points in the southern Caddo area</span>.”</span> Journal Article. <em>Open Science Framework</em>. <a href="https://doi.org/10.17605/OSF.IO/DEJ74">https://doi.org/10.17605/OSF.IO/DEJ74</a>.
</div>
<div id="ref-RN21001" class="csl-entry">
Selden Jr., Robert Z., John E. Dockall, and Morgane Dubied. 2020. <span>“A Quantitative Assessment of Intraspecific Morphological Variation in Gahagan Bifaces from the Southern Caddo Area and Central Texas.”</span> <em>Southeastern Archaeology</em> 39 (2): 125–45. <a href="https://doi.org/10.1080/0734578x.2020.1744416">https://doi.org/10.1080/0734578x.2020.1744416</a>.
</div>
<div id="ref-RN11783" class="csl-entry">
Selden Jr., Robert Z., John E. Dockall, and Harry J. Shafer. 2018. <span>“<span class="nocase">Lithic Morphological Organisation: Gahagan Bifaces from the Southern Caddo Area</span>.”</span> <em>Digital Applications in Archaeology and Cultural Heritage</em> 10: e00080. <a href="https://doi.org/10.1016/j.daach.2018.e00080">https://doi.org/10.1016/j.daach.2018.e00080</a>.
</div>
<div id="ref-RN479" class="csl-entry">
Sherratt, Emma, David J. Gower, Christian Peter Klingenberg, and Mark Wilkinson. 2014. <span>“<span class="nocase">Evolution of Cranial Shape in Caecilians (Amphibia: Gymnophiona)</span>.”</span> <em>Evolutionary Biology</em> 41 (4): 528–45. https://doi.org/<a href="https://doi.org/10.1007/s11692-014-9287-2">https://doi.org/10.1007/s11692-014-9287-2</a>.
</div>
<div id="ref-RN11563" class="csl-entry">
Slice, Dennis E. 2001. <span>“<span class="nocase">Landmark Coordinates Aligned by Procrustes Analysis Do Not Lie in Kendall’s Shape Space</span>.”</span> <em>Systematic Biology</em> 50 (1): 141–49. <a href="https://doi.org/10.1080/10635150119110">https://doi.org/10.1080/10635150119110</a>.
</div>
<div id="ref-RN5694" class="csl-entry">
Zelditch, Miriam Leah, Donald L. Swiderski, H. David Sheets, and William L. Fink. 2004. <em>Geometric Morphometrics for Biologists : A Primer</em>. Book. Burlington: Elsevier Science. <a href="http://ebookcentral.proquest.com/lib/tamucs/detail.action?docID=298308">http://ebookcentral.proquest.com/lib/tamucs/detail.action?docID=298308</a>.
</div>
</div>
<div class="footnotes">
<hr />
<ol start="1">
<li id="fn1"><p>Heritage Research Center, Stephen F. Austin State University; Department of Biology, Stephen F. Austin State University; and Cultural Heritage Department, Jean Monnet University, <a href="mailto:zselden@sfasu.edu" class="email">zselden@sfasu.edu</a><a href="#fnref1" class="footnote-back">↩︎</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>

<a href="linear-shapesize.html" class="navigation navigation-next navigation-unique" aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "sepia",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/USERNAME/REPO/edit/BRANCH/index.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["_main.pdf", "_main.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
